packages/models/src/events/events.ts

```diff
@@ -9,6 +9,7 @@ export const EventEnvelope = <TEventZodType extends z.ZodType>(
       sequence_num: z.number(),
       stream_id: z.string().uuid(),
       version: z.number(),
+      correlation_id: z.string().optional(),
```

Comment:
is it possible to remove optionality?

Reply:
In theory, the database doesn't define this column as `NOT NULL` so this field should be optional. We could decide that we can initialize this field with a new uuid if it's not present in the database but I prefer not to hide this information

Reply:
fair point.
I was thinking that for v2 events the correlation id is mandatory. 
Maybe we could think about a versioning of the `EventEnvelope`, but not now